pkg/cnab/cnab-to-oci/registry.go

```diff
@@ -114,6 +114,36 @@ func (r *Registry) PushBundle(bun bundle.Bundle, tag string, insecureRegistry bo
 	return &rm, nil
 }
 
+// Pushes a bundle while using a relocationMap to correctly locate copied images. Identical to above, except for remotes.WithRelocationMap(reloMap)
+func (r *Registry) PushBundleWithRelocationMap(bun bundle.Bundle, tag string, reloMap relocation.ImageRelocationMap, insecureRegistry bool) (*relocation.ImageRelocationMap, error) {
```

Comment:
There are three cases where we call the original function `PushBundle`:

1. `porter publish`, this doesn't have a relocation map.
2. `porter publish --archive` I have pulled in @vdice to get more context on this one.
3. `porter copy`, which is what you are fixing.

Porter does not have an API compatibility guarantee at the code level, so we are free to edit the signature of the original function. Would you please update the original function `PushBundle` with the additional parameter `reloMap relocation.ImageRelocationMap`. In case #1 above, since we don't have one, we can pass in an empty relocation map.